images/win/scripts/Installers/Install-ActionArchiveCache.ps1

```diff
@@ -0,0 +1,20 @@
+################################################################################
+##  File:  Install-ActionArchiveCache.ps1
+##  Desc:  Install Action Archive Cache
+##         from https://github.com/actions/action-versions
+################################################################################
+
+if (-not (Test-Path $env:ACTIONS_RUNNER_ACTION_ARCHIVE_CACHE))
+{
+    Write-Host "Creating action archive cache folder"
+    New-Item -ItemType Directory -Path $env:ACTIONS_RUNNER_ACTION_ARCHIVE_CACHE | Out-Null
+}
+
+Write-Host "Download Latest action-versions archive"
+$latestRelease = Invoke-RestMethod -Uri "https://api.github.com/repos/actions/action-versions/releases/latest"
```

Comment:
use `Start-DownloadWithRetry` which is defined here https://github.com/actions/runner-images/blob/a4234d46c7b5db3e2187af0db44790e1a898c802/images/macos/helpers/Common.Helpers.psm1#L123